meal-sharing-final/backend/Reviews.cs

```diff
@@ -0,0 +1,60 @@
+using System.Text.Json.Serialization;
+using Dapper;
+using MySql.Data.MySqlClient;
+
+namespace HackYourFuture.Week7;
+
+public interface IReviewRepository
+{
+    Task<IEnumerable<Review>> GetAllReviews();
+    Task<IEnumerable<Review>> ReviewMealWithId(int id);
+    Task<Review> AddReview(Review review);
+
+}
+
+public class ReviewRepository : IReviewRepository
+{
+    private string connectionString;
+
+    public ReviewRepository(IConfiguration configuration)
+    {
+        this.connectionString = configuration.GetConnectionString("Default");
+    }
+
+    public async Task<IEnumerable<Review>> GetAllReviews()
+    {
+        using var connection = new MySqlConnection(connectionString);
+        var reviews = await connection.QueryAsync<Review>("SELECT * FROM Review");
+        return reviews;
+    }
+
+    public async Task<IEnumerable<Review>> ReviewMealWithId(int id)
```

Comment:
maybe a more suggestive naming for this method could be `MealReviews(int mealId)` or `ReviewsForMeal(int mealId)`